File: py/solns/removeDuplicatesFromSortedArr/removeDuplicatesFromSortedArr.py
class Solution:
    @staticmethod
    def naive(nums):
        if len(nums)<=1: return len(nums)
        maxVal = nums[0]
        ptid = 1
        for i in range(1,len(nums)):
            if nums[i]>maxVal: 
                maxVal = nums[i]
                nums[i],nums[ptid]=nums[ptid],nums[i]
                ptid+=1
        return ptid

File: cpp/example/src/RemoveDuplicatesFromSortedArr/removeDuplicatesFromSortedArr.cpp
#include "removeDuplicatesFromSortedArr.h"

int removeDuplicatesFromSortedArr::naive(vector<int> nums){
    if (nums.size()<=1) return nums.size();
    int maxVal = nums[0];
    int ptid = 1;
    for (int i=1;i<nums.size();i++){
        if (nums[i]>maxVal){
            maxVal = nums[i];
            int tmp = nums[i];
            nums[i] = nums[ptid];
            nums[ptid] = tmp;
            ptid += 1;
        }
    }
    return ptid;
}